doc/classes/Viewport.xml

```diff
@@ -128,6 +128,16 @@
 				Returns the [Control] having the focus within this viewport. If no [Control] has the focus, returns null.
 			</description>
 		</method>
+		<method name="gui_get_mouse_over" qualifiers="const">
+			<return type="Control" />
+			<description>
+				Returns the [Control] that the mouse is currently hovering over in this viewport. 
+				If no [Control] has the cursor, returns null.
+
+				This value is always [Control] which is the has the mouse, typically the leaf node or deepest level of the tree which claims focus.
+				This is very useful when used together with [method Node.is_ancestor_of] to find if the mouse is within a control group.
```

Comment:
```suggestion
				Returns the [Control] that the mouse is currently hovering over in this viewport. If no [Control] has the cursor, returns null.
				This value is always [Control] which is the has the mouse, typically the leaf node or deepest level of the tree which claims focus. This is very useful when used together with [method Node.is_ancestor_of] to find if the mouse is within a control group.
```
Each separate line is a new paragraph, this is much more compact, otherwise it would be very spread out

I'm not entirely sure about the wording of "group" as it might be confused with the existing concept of groups (which it isn't here) but can't think of a different wording